### DISCUSSION TITLE
Unable to understand some concepts and "some explainations"

### DISCUSSION THREAD

**Discussion by printfdebugging** [RayTracing/raytracing.github.io#1562] [Q&A] [answered]
Hi
in the section 9.4 it is written
> "This means that for diffuse objects, they will appear darker because less light bounces toward the camera. For the shadows, more light bounces straight-up, so the area underneath the sphere is darker."

I am not getting the "less light bounces towards the camera... For shadows ..." Doesn't the are underneath the sphere appear darker because the rays keep bouncing between 2 spheres more than `max_depth` times? I even doubt what I just reasoned.

**Comment by hollasch** [comment, marked as answer]
> Doesn't the are underneath the sphere appear darker because the rays keep bouncing between 2 spheres more than max_depth times?

Yeah, that's one way to think about it. The complementary thought is that in shadowed regions, more of the background/sky is occluded by nearby objects. Also, even if the ray bounces fewer than `max_depth` times, its contributed light is diminished with each bounce.
